### PR TITLE
Rework blindness, UTs for speech.

### PR DESCRIPTION
**Pull request by Sindorman**
In this PR I will rework vision to have API for species and add UTs for vision and speech.
TO-DO:

- [x] Add UTs for speech to test speech disabilities

- [x] make vision disability into species API 

- [x] Rework `say()` and `hear_say()` to not redundantly check if mobs can hear, will increase performance.

### CODE REVIEW

**Comment by Sindorman** [comment]
!review

**Comment by skull132** [comment]
It was merged. This PR accomplishes the same thing as the other one. Though with this one I'd also like Poze to implement UTs, to make sure this doesn't happen in the future.

**Comment by Sindorman** [comment]
yes, This is going WIP. I am on it

**Comment by Sindorman** [comment]
accidentally closed it by deleting the branch while cleaning up branches

**Comment by skull132** [comment]
Marked WIP, no movement for a month and more. Closing.

File: aurorastation.dme [hunk 1]
```diff
@@ -51,6 +51,7 @@
 #include "code\__defines\tgs.dm"
 #include "code\__defines\time.dm"
 #include "code\__defines\turfs.dm"
+#include "code\__defines\unit_testing.dm"
```
Comment (Karolis2011) [inline aurorastation.dme]: This should never been commited

File: code/__defines/unit_testing.dm [hunk 2]
```diff
@@ -7,4 +7,4 @@
  *   Should you wish to edit set UNIT_TEST to 1 like so:
  *   #define UNIT_TEST 1
  */
-#define UNIT_TEST 0
+#define UNIT_TEST 1
```
Comment (Karolis2011) [inline code/__defines/unit_testing.dm]: This should never been commited

Comment (Sindorman) [reply code/__defines/unit_testing.dm]: Oh right, forgot to take it out. Thank you

**Comment by Sindorman** [comment]
Both hearing and speech

**Pull request merged**